src/js/view/utils/resize-listener.js

```diff
@@ -1,24 +1,51 @@
 import { requestAnimationFrame, cancelAnimationFrame } from 'os/utils/request-animation-frame';
 import { createElement } from 'os/utils/dom';
+import { css, style } from 'os/utils/css';
 
 export default class ResizeListener {
 
     constructor(element, callback) {
-        const hiddenHtml = '<div class="jw-resize-trigger"><div class="jw-expand-trigger">' +
-            '<div style="height:1px;"></div></div><div class="jw-contract-trigger"></div></div>';
-        const hiddenElement = createElement(hiddenHtml);
-        const expandElement = hiddenElement.firstChild;
+        const topLeft = {
+            display: 'block',
+            position: 'absolute',
+            top: 0,
+            left: 0
+        };
+
+        const stretch = {
+            width: '100%',
+            height: '100%'
+        };
+
+        css('.jw-contract-trigger::before', Object.assign({
```

Comment:
Since `.jw-contract-trigger` and these other styles would theoretically only exist in this form of CSS-in-JS, any static analysis tools used for the CSS output can be rendered inaccurate.